browser-client.cpp

```diff
@@ -206,13 +206,13 @@ bool BrowserClient::OnProcessMessageReceived(
 		if (name == "getScenes") {
 			struct obs_frontend_source_list list = {};
 			obs_frontend_get_scenes(&list);
-			std::vector<const char *> scenes_vector;
+			std::vector<Json> scenes_vector;
 			for (size_t i = 0; i < list.sources.num; i++) {
 				obs_source_t *source = list.sources.array[i];
 				scenes_vector.push_back(
 					obs_source_get_name(source));
 			}
-			json = scenes_vector;
+			json = Json::array(scenes_vector);
```

Comment:
nlohmann is smart enough to convert a vector of JSONs to a JSON array of objects. And `Json::array` is json11 only.
```suggestion
			json = scenes_vector;
```

Reply:
Confirmed. Scrapped the change on my side.